files/zh-cn/mdn/community/mdn_content/index.md

```diff
@@ -1,49 +1,47 @@
 ---
-title: 改正内容错误
+title: 为 MDN Web 文档内容做出贡献
 slug: MDN/Community/MDN_content
 ---
 
 {{MDNSidebar}}
 
-MDN 文档的问题被报告在 [content 仓库的 issues](https://github.com/mdn/content/issues)中。本文将帮助你，根据你的专业知识和空闲时间来找到最适合解决的问题，并且讲述解决这些问题的主要步骤。
+MDN Web 文档的问题在[内容仓库的议题](https://github.com/mdn/content/issues)中报告。本文将帮助你根据你的专业知识和空闲时间来找到*最适合解决*的议题，并概述解决这些议题的主要步骤。
 
-> **备注：** 我们有很多内容上的问题——非常感谢你在解决这些问题时给予的任何帮助！
+> **备注：** 我们有很多内容上的错误——非常感谢你在解决这些错误时给予的任何帮助！
 
-## 我们需要什么帮助？
+## 我们需要什么帮助
 
-为了帮助你选择要处理的内容问题，我们使用 GitHub 标签对它们进行了排序。
+为了帮助你选择要处理的内容议题，我们使用 GitHub 标签对它们进行了排序。
 
 下面的标签可以帮助你根据空闲时间来选择任务。
 
-| 标签                                                                                                                           | 描述                             |
-| ------------------------------------------------------------------------------------------------------------------------------ | -------------------------------- |
-| [内容仓库中花费不到 30 分钟的任务](https://github.com/mdn/content/issues?q=is%3Aissue+is%3Aopen+label%3A%22time%3A+-30mins%22) | 可能需要花费不到 30 分钟的任务。 |
-| [内容仓库中花费不到 3 小时的任务](https://github.com/mdn/content/issues?q=is%3Aissue+is%3Aopen+label%3A%22time%3A+-3hr%22)     | 可能需要花费不到 3 小时的任务。  |
-| [内容仓库中花费不到 2 天的任务](https://github.com/mdn/content/issues?q=is%3Aissue+is%3Aopen+label%3A%22time%3A+-2days%22)     | 可能需要花费不到 2 天的任务。    |
+| 标签                                                                       | 描述                                                                         |
+| -------------------------------------------------------------------------- | ---------------------------------------------------------------------------- |
+| [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | 可能只需耗费一点精力的任务，例如修复拼写错误或添加几句用于澄清某一事实的话。 |
+| [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | 可能需要付出不小的精力的任务。例如，更新页面以反映正式规范中的变更。         |
+| [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | 可能需要耗费大量精力的任务，例如撰写新的指南和参考页。                       |
```

Comment:
should we use `translated-content` for these links?

Reply:
If we refer to `translated-content`, we have to modify most content in this document. As we won't label issue by technologies.